lib/node_modules/@stdlib/math/base/special/bessely0/benchmark/python/benchmark.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""Benchmark bessely0."""
+
+from __future__ import print_function
+import timeit
+
+NAME = "bessely0"
```

Comment:
The name, however, stays the same, as the name is what we use to group benchmarks across languages together.